src/public/main.js

```diff
@@ -62,3 +63,13 @@ $(document).ready(() => {
     }, 500)
   })
 })
+
+// url hash for direct links to subtabs on inauguration.html
+$(document).ready(() => {
+  if (location.hash) {
+    $("a[href='" + location.hash + "']").tab('show')
+  }
+  $('.nav').on('click', 'a[data-toggle]', function onClickGethref(event) {
+    location.hash = this.getAttribute('href')
+  })
+})
```

Comment:
for now, we're using plain-old html as our 'bundler'. since this is specific to inauguration.html, should we just put it in a script tag on that page?